README.md

```diff
@@ -112,9 +112,16 @@ BibTeX entry:
 }
 ```
 
+## Trane Code
```

Comment:
Can we move this to the section History right after where it says "This repository is a second iteration..."  Perhaps it does not require a sub heading - Trane code